config/preview/config.yaml

```
@@ -1,6 +1,6 @@
 baseURL: https://docs-staging.datadoghq.com/$CI_COMMIT_REF_NAME/
 buildfuture: true
-builddrafts: true
+builddrafts: false
```

```suggestion
builddrafts: true
```

Apply this suggestion before merging. I temporarily disabled builddrafts on preview so we could test the page is respecting the `draft: true` frontmatter. We want it back the way it was on release.